src/JuMP.jl

```diff
@@ -450,6 +450,15 @@ function solve_time(model::Model)
     return MOI.get(model, MOI.SolveTime())
 end
 
+"""
+    set_parameter(model::Model, name, value)
+
+Sets solver-specific parameter identified by `name` to `value`.
+"""
+function set_parameter(model::Model, name, value)
```

Comment:
Add it do the documentation besides `set_silent`

Reply:
Done.